src/globus_cli/commands/streams/tunnel/update.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+import uuid
+
+import click
+import globus_sdk
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import OMITTABLE_STRING, command
+from globus_cli.termio import display
+
+from ._common import TUNNEL_STANDARD_FIELDS
+
+
+@command("create", short_help="Create a Globus Tunnel.")
+@click.argument("tunnel_id", metavar="TUNNEL_ID", type=click.UUID)
+@click.option(
+    "--label",
+    type=OMITTABLE_STRING,
+    default=globus_sdk.MISSING,
+    help="A friendly label to associate with the Tunnel.",
+)
+@LoginManager.requires_login("auth", "transfer")
+def update_tunnel_command(
+    login_manager: LoginManager,
+    *,
+    tunnel_id: uuid.UUID,
+    label: str | globus_sdk.MissingType,
+) -> None:
+    """
+    Create a Globus Tunnel.
+    """
+    tunnel_client = login_manager.get_transfer_client()
+    data = {
+        "data": {
+            "type": "Tunnel",
+            "attributes": {
+                "label": label,
+            },
+        }
+    }
```

Comment:
If `label` is `None` does this erase the label? Is this intended behavior if the user runs

```
globus tunnel update $TUNNEL_ID
```

and provides nothing else? I would expect that if the user doesn't provide a label option then the label wouldn't get overwritten.